dlp-webapp/src/main/resources/locale/notification/template/Notification_en.properties

```diff
@@ -15,21 +15,21 @@ Notification.mail.DlpDetectedItemPlugin.footer=eXo Platform
 UINotification.label.DlpUserDetectedItemPlugin=Detected item for users
 UINotification.title.DlpUserDetectedItemPlugin=Item needs to be analyzed
 Notification.subject.DlpUserDetectedItemPlugin=Item has been moved in quarantine
-Notification.intranet.message.DlpUserDetectedItemPlugin=This file is under quarantine to be analyzed
-
 #############################################################################
 #                         DlpAdminDetectedItemPlugin                        #
 #############################################################################
 UINotification.label.DlpAdminDetectedItemPlugin=Detected item for admins
 UINotification.title.DlpAdminDetectedItemPlugin=Item has been moved in quarantine, waiting for your action
 Notification.subject.DlpAdminDetectedItemPlugin=Item has been moved in quarantine
-Notification.intranet.message.DlpAdminDetectedItemPlugin=This file is in quarantine waiting for your action
-
 #############################################################################
 #                         DlpUserRestoredItemPlugin                         #
 #############################################################################
 UINotification.label.DlpUserRestoredItemPlugin=Restored item
 UINotification.title.DlpUserRestoredItemPlugin=Item has been validated
 Notification.title.DlpUserRestoredItemPlugin=Item has been validated
 Notification.subject.DlpUserRestoredItemPlugin=Item has been validated
-Notification.intranet.message.DlpUserRestoredItemPlugin=This file has been validated and restored to its location after a quarantine phase
+####
+Notification.label.types.quarantine=Quarantine
+Notification.message.DlpUserRestoredItemPlugin=This file has been validated and restored to its location after a quarantine phase
+Notification.message.DlpAdminDetectedItemPlugin=This file is in quarantine waiting for your action
+Notification.message.DlpUserDetectedItemPlugin=This file is under quarantine to be analyzed
```

Comment:
Why do you change the key here ? 
Changing the key means that we need to re-translate it in crowdin. 
Is it possible to keep the same key ?